tko_web_sessions_management/main.py

```diff
@@ -237,10 +239,10 @@ def save_session(
                 'unsuccessful_message': unsuccessful_message,
             }
             # create session record for unsuccessful login with sudo
-            if uid == SUPERUSER_ID:
-                session_obj.sudo().create(values)
-            else:
-                session_obj.create(values)
+#             if uid == SUPERUSER_ID:
+#                 session_obj.sudo().create(values)
+#             else:
+            session_obj.sudo().create(values)
```

Comment:
I am not sure but this line was restricting immediate update of session lines in case of session out and login. 
I tried this way and found it working.